.github/workflows/release.yaml

```diff
@@ -1,53 +0,0 @@
-name: Publish Python Package
-
-on:
-  release:
-    types: [created]
-  workflow_run:
-    workflows: ["Run Tests"] 
-    types:
-      - completed
-    branches:
-      - master
-    
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
In the remaining workflow where we are running the tests I have not bothered implementing a test matrix. Although we are using pytest in theory, none of our tests are actually running python package code. They're just shelling out to `lighthouse`. If we ever add some tests that are exercising python code, we can add a matrix there.